cmd/topo-generator/topo-generator.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
Licenses are missing. Add Intel one like this: https://github.com/onosproject/wcmp-app/blob/master/pkg/controller/connection/controller.go#L1